BackEndApp/v1/views.py

```diff
@@ -501,15 +501,15 @@ def download_code(request):
 def get_users(request):
     try:
         username = request.data.get("username")
-        user = User(username=username, password=None)
+        user = User(username="asd", password=None)
         user = user.find()
-
-        users = os.listdir(os.path.expanduser("~") + "\.autodl")
+        users = Store(user)
```

Comment:
```suggestion
        store = Store(user)
```

Naming of variables is one of the hardest things in software engineering, users = Store(user) doesn't sound right now, does it?